Gemfile

```diff
@@ -23,7 +24,7 @@ gem 'rails', '4.2.5.1'
 gem 'redis', '3.2.1', require: ['redis', 'redis/connection/hiredis']
 gem 'remote_syslog_logger', '1.0.3'
 gem 'sidekiq', '4.1.4'
-gem 'sidekiq-pro', '3.3.1'
+#gem 'sidekiq-pro', '3.3.1'
```

Comment:
Ok I will remove them and push to microservice template repo

On Thu, Sep 8, 2016 at 10:48 PM, Phoenix Chu <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/johnnieo/Micro-service-templates/pull/1#discussion_r78049799>
> :
>
> > @@ -23,7 +24,7 @@ gem 'rails', '4.2.5.1'
> >  gem 'redis', '3.2.1', require: ['redis', 'redis/connection/hiredis']
> >  gem 'remote_syslog_logger', '1.0.3'
> >  gem 'sidekiq', '4.1.4'
> > -gem 'sidekiq-pro', '3.3.1'
> > +#gem 'sidekiq-pro', '3.3.1'
>
> same here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/johnnieo/Micro-service-templates/pull/1/files/1334ec886a748fee8e5a20f3bd20bd2be7ad798b#r78049799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUCCBc8TS7hpP9sbKUZrAIqXq2iMIyKCks5qoENYgaJpZM4J3vgo>
> .
>



-- 
Thanks & Regards,
*Kalpesh Munot*
Senior Ruby on Rails Developer